src/System.Security.Cryptography.Xml/src/project.json

```diff
@@ -7,7 +7,7 @@
     },
     "net463": {
```

Comment:
@bartonjs I left this as net463 instead of net462 as the rest of the projects are configured this way.